### PR TITLE
Backend/feature/poll closing & grading

### PR DESCRIPTION
**Pull request by EmreBatuhan**
This pr implements the close Poll routine and adds poll closing and grading mechanics for discrete polls and continuous poll. Also domain points are updated according to poll grading implemented.

### CODE REVIEW

File: prediction-polls/backend/src/routines/pollCloseRoutine.js [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+    await pollDB.updateJuryReward(pollObject.id,new_jury_reward)
+    await modDB.updateJuryRewardforRequests(pollObject.id,new_jury_reward)
+
+}
+
+function handleNoDueDatePoll(){
+     // 
+}
+
+cron.schedule('*/60 * * * * *', closeFinishedPolls);
```
Comment (Sefik-Palazoglu) [inline prediction-polls/backend/src/routines/pollCloseRoutine.js:255]: does this work? https://www.npmjs.com/package/node-cron#allowed-values says allowed values are 0-59. If it works keep it.

Comment (EmreBatuhan) [reply prediction-polls/backend/src/routines/pollCloseRoutine.js:255]: I wanted to discuss the routine time intervals with you so they are open to change

**Review by Sefik-Palazoglu — COMMENTED** [review]
Looks good, check my comment in pollCloseRoutine.js, then I can approve.

**Review by Sefik-Palazoglu — APPROVED** [review]
LGTM!

**Review by AliNasra — APPROVED** [review]
Great work, Batu! The period of activating changes might be changed later.

**Pull request merged**